erc20/alias-erc20-deploy.js

```diff
@@ -54,48 +54,48 @@ async function deploy(info) {
+    await conf.child.newInstanceBridge.methods.registerOperator(bridge.child.operator).send({ from: conf.sender.child.address, gas: 100000000, value: 0 });
+    await conf.parent.newInstanceBridge.methods.registerOperator(bridge.parent.operator).send({ from: conf.sender.parent.address, gas: 100000000, value: 0 });
+
+    const alias = "MYBRIDGE2"
+    const url = bridge.child.url;
+    log = 'registering bridges to the child node'
+    await jsonRpcReq(url, log, 'subbridge_registerBridgeByAlias', [alias, conf.contract.child.bridge, conf.contract.parent.bridge]);
+
+    log = 'subscribing bridges to the child node'
+    await jsonRpcReq(url, log, 'subbridge_subscribeBridgeByAlias', [alias]);
+
+    log = 'register token to subbridge..'
+    await jsonRpcReq(url, log, 'subbridge_registerTokenByAlias', [alias, conf.contract.child.token, conf.contract.parent.token]);
+  }
+
+  // setOperatorThreshold
+  await conf.contract.child.newInstanceBridge.methods.setOperatorThreshold(conf.bridges.length).send({ from: conf.sender.child.address, gas: 100000000, value: 0 });
+  await conf.contract.parent.newInstanceBridge.methods.setOperatorThreshold(conf.bridges.length).send({ from: conf.sender.parent.address, gas: 100000000, value: 0 });
 
   // transferOwnership
-  await conf.child.newInstanceBridge.methods.transferOwnership(conf.child.operator).send({ from: conf.child.sender, gas: 100000000, value: 0 });
-  await conf.parent.newInstanceBridge.methods.transferOwnership(conf.parent.operator).send({ from: conf.parent.sender, gas: 100000000, value: 0 });
+  await conf.contract.child.newInstanceBridge.methods.transferOwnership(conf.bridges[0].child.operator).send({ from: conf.sender.child.address, gas: 100000000, value: 0 });
+  await conf.contract.parent.newInstanceBridge.methods.transferOwnership(conf.bridges[0].parent.operator).send({ from: conf.sender.parent.address, gas: 100000000, value: 0 });
 
   const filename  = "transfer_conf.json"
   fs.writeFile(filename, JSON.stringify(conf), (err) => {
       if (err) {
           console.log("Error:", err);
       }
   })
-  
-  const alias = "MYBRIDGE2"
-  const url = conf.child.url;
-  log = 'registering bridges to the child node'
-  await jsonRpcReq(url, log, 'subbridge_registerBridgeByAlias', [alias, conf.child.bridge, conf.parent.bridge]);
-
-  log = 'subscribing bridges to the child node'
-  await jsonRpcReq(url, log, 'subbridge_subscribeBridgeByAlias', [alias]);
-
-  log = 'register token to subbridge..'
-  await jsonRpcReq(url, log, 'subbridge_registerTokenByAlias', [alias, conf.child.token, conf.parent.token]);
-
-  /*
-   * Initialize service chain configuration with three logs via interaction with attached console
-  console.log(`subbridge.registerBridgeByAlias("${alias}", "${conf.child.bridge}", "${conf.parent.bridge}")`)
-  console.log(`subbridge.subscribeBridgeByAlias("${alias}")`)
-  console.log(`subbridge.registerTokenByAlias("${alias}", "${conf.child.token}", "${conf.parent.token}")`)
-  */
```

Comment:
Which fact did you lead to erasing this code?

Reply:
@hyunsooda It was moved into the loop located a few lines above. If HA configured, we have to call `registerBridge`, `subscribeBridge` and `registerToken` in all configured bridges.

Reply:
Gotcha. Could you update the comments inside the loop? Those are a manual way to setup the required initialization steps and are referenced from [doc](https://ko.docs.klaytn.com/node/service-chain/getting-started/value-transfer).

Reply:
This PR will be closed and this review is resolved in the followed PR #8. Thus marking this review resolved.